examples/uct_hello_world.c

```diff
@@ -346,19 +346,24 @@ static ucs_status_t dev_tl_lookup(const cmd_args_t *cmd_args,
         for (md_index = 0; md_index < component_attr.md_resource_count; ++md_index) {
             status = uct_md_config_read(components[cmpt_index], NULL, NULL,
                                         &md_config);
-            CHKERR_JUMP(UCS_OK != status, "read MD config",
-                        release_component_list);
+            if (UCS_OK != status) {
```

Comment:
Are you sure we need to modify this flow and the code after `uct_md_query`?
Isn't it enough to only change the logic for `uct_md_open`?